### PR TITLE
Feature/day 2 assignment

### PR DESCRIPTION
**Pull request by ayushpandey-successive**
Programs using Arrow Functions.

1. Write a program to give a random output between 1 to 10 (AC: 2 simultaneous answers could not be same)
ques-1

2. Write a function to take input(age, name, phone no.) from user and display it
ques-2

3. Write a function expression that takes in another function as an argument
ques-3

4. Write a function expression that takes in a number and returns its square.
ques-4

5. Write a function expression that takes in two numbers and returns their sum.
ques-5

6. Write a function expression that takes in a number and returns true if it's even and false if it's odd.
ques-6

7. All of the above questions from 3-6 needs to be done with arrow functions also
ques-7

### CODE REVIEW

**Comment by saurabh-yadav-successive** [comment]
Please add descriptions and code execution SS in PR

File: day-2/ques-1.js [hunk 1]
```diff
@@ -0,0 +1,4 @@
+
```
Comment (saurabh-yadav-successive) [inline day-2/ques-1.js:2]: Add problem statement comment

File: day-2/ques-2.js [hunk 2]
```diff
@@ -0,0 +1,5 @@
+let username="Ayush Pandey"
```
Comment (saurabh-yadav-successive) [inline day-2/ques-2.js]: Add problem statement comment

File: day-2/ques-1.js [hunk 3]
```diff
@@ -0,0 +1,4 @@
...
+
+let randomnumber=()=> Math.ceil(Math.random()*10);
```
Comment (saurabh-yadav-successive) [inline day-2/ques-1.js]: method name should be in camel case format.

File: day-2/ques-2.js (hunk 2 above)
Comment (saurabh-yadav-successive) [inline day-2/ques-2.js]: why we defiine variable in let, if variable value is not changing.

File: day-2/ques-3.js [hunk 4]
```diff
@@ -0,0 +1,10 @@
+// Write a function expression that takes in another function as an argument
+
+let sum=(a,b)=>a+b;
```
Comment (saurabh-yadav-successive) [inline day-2/ques-3.js]: method should be define in const

File: day-2/ques-3.js [hunk 5]
```diff
@@ -0,0 +1,10 @@
...
+
+let average=(a,b,sum)=>{
+    let s=sum(a,b);
+    return s/2;
+}
+
```
Comment (saurabh-yadav-successive) [inline day-2/ques-3.js]: remove extra spaces between two staements

File: day-2/ques-4.js [hunk 6]
```diff
@@ -0,0 +1,11 @@
+//Write a function expression that takes in a number and returns its square.
+
```
Comment (saurabh-yadav-successive) [inline day-2/ques-4.js:2]: Remove extra space.

File: day-2/ques-4.js [hunk 7]
```diff
@@ -0,0 +1,11 @@
+//Write a function expression that takes in a number and returns its square.
```
Comment (saurabh-yadav-successive) [inline day-2/ques-4.js]: Problem statement should be start with Problem Statement: then start

File: day-2/ques-4.js [hunk 8]
```diff
@@ -0,0 +1,11 @@
...
+
+
+function squareofnumber (a)
+{
+    return a*a;
+}
+
+
```
Comment (saurabh-yadav-successive) [inline day-2/ques-4.js]: remove extra spaces

File: day-2/ques-4.js [hunk 9]
```diff
@@ -0,0 +1,11 @@
+//Write a function expression that takes in a number and returns its square.
+
+
+function squareofnumber (a)
```
Comment (saurabh-yadav-successive) [inline day-2/ques-4.js]: function name should be in camel case format

File: day-2/ques-5.js [hunk 10]
```diff
@@ -0,0 +1,7 @@
+// Write a function expression that takes in two numbers and returns their sum.
```
Comment (saurabh-yadav-successive) [inline day-2/ques-5.js]: Correct problem statement format as mentioned above

File: day-2/ques-6.js [hunk 11]
```diff
@@ -0,0 +1,9 @@
+//Write a function expression that takes in a number and returns true if it's even and false if it's odd.
```
Comment (saurabh-yadav-successive) [inline day-2/ques-6.js]: Correct problem statement format as mentioned above

File: day-2/ques-2.js [hunk 12]
```diff
@@ -0,0 +1,5 @@
...
+let age=21
+let phonenumber=9569452271
```
Comment (saurabh-yadav-successive) [inline day-2/ques-2.js]: variable name shoul be in camel case

File: day-2/ques-7.js [hunk 13]
```diff
@@ -0,0 +1,38 @@
+//7. All of the above questions from 3-6 needs to be done with arrow functions also
+
+////////////////////////////////
+//question 3
+
+let sum=(a,b)=>a+b;
```
Comment (saurabh-yadav-successive) [inline day-2/ques-7.js]: variable should be a const

File: day-2/ques-7.js [hunk 14]
```diff
@@ -0,0 +1,38 @@
...
+}
+
+console.log(average(1,2,sum))
+
+
+////////////////////////////////
+
+//question 4
+
+let squareofnumber = (a)=> a*a;
```
Comment (saurabh-yadav-successive) [inline day-2/ques-7.js]: variable name in camel case

File: day-2/ques-2.js (hunk 2 above)
Comment (ayushpandey-successive) [reply day-2/ques-2.js]: resolved

File: day-2/ques-3.js (hunk 4 above)
Comment (ayushpandey-successive) [reply day-2/ques-3.js]: resolved

File: day-2/ques-5.js (hunk 10 above)
Comment (ayushpandey-successive) [reply day-2/ques-5.js]: resolved

File: day-2/ques-6.js (hunk 11 above)
Comment (ayushpandey-successive) [reply day-2/ques-6.js]: resolved

File: day-2/ques-2.js (hunk 12 above)
Comment (ayushpandey-successive) [reply day-2/ques-2.js]: resolved

File: day-2/ques-7.js (hunk 13 above)
Comment (ayushpandey-successive) [reply day-2/ques-7.js]: resolved

File: day-2/ques-7.js (hunk 14 above)
Comment (ayushpandey-successive) [reply day-2/ques-7.js]: resolved

File: day-2/ques-1.js (hunk 3 above)
Comment (ayushpandey-successive) [reply day-2/ques-1.js]: resolved

File: day-2/ques-2.js (hunk 2 above)
Comment (ayushpandey-successive) [reply day-2/ques-2.js]: resolved

File: day-2/ques-3.js (hunk 5 above)
Comment (ayushpandey-successive) [reply day-2/ques-3.js]: resolved

File: day-2/ques-4.js (hunk 6 above)
Comment (ayushpandey-successive) [reply day-2/ques-4.js:2]: resolved

File: day-2/ques-4.js (hunk 7 above)
Comment (ayushpandey-successive) [reply day-2/ques-4.js]: resolved

File: day-2/ques-4.js (hunk 8 above)
Comment (ayushpandey-successive) [reply day-2/ques-4.js]: resloved

File: day-2/ques-4.js (hunk 9 above)
Comment (ayushpandey-successive) [reply day-2/ques-4.js]: resolved

**Review by saurabh-yadav-successive — APPROVED** [review]

**Pull request merged**